packages/issuer-sdk-js/package.json

```
@@ -26,6 +26,10 @@
     "./human": {
       "import": "./dist/human.js",
       "types": "./dist/types/human.d.ts"
+    },
+    "./credentilas": {
+      "import": "./dist/credentilas.js",
```

Typo on the file name:
```suggestion
      "import": "./dist/credentials.js",
```